Hummingbird.docc/Tutorials/MongoKitten/MongoKitten-1-Setup.tutorial

```diff
@@ -0,0 +1,139 @@
+                
+                Add the MongoKitten dependency, and modify the `AppArguments` to contain two new variables.
+                @Code(name: "Sources/App/Application+build.swift", file: mongokitten-setup-06.swift, previousFile: mongokitten-setup-05.swift)
+            }
+            
+            @Step {
+                Open `Sources/App/App.swift`
+                
+                This contains an `App` type conforming to `AsyncParsableCommand` with three options, the `hostname` and `port` are used to define the server bind address, `logLevel` sets the level of logging required. Finally the `run()` function which calls `buildApplication(_:)` to create an `Application` and then runs it using `runService()`. You can find out more about the argument parser library [here](https://apple.github.io/swift-argument-parser/documentation/argumentparser).
+                @Code(name: "Sources/App/App.swift", file: mongokitten-setup-07.swift)
+            }
+            @Step {
+                Add the new app arguments with default values.
+                
+                @Code(name: "Sources/App/App.swift", file: mongokitten-setup-08.swift)
+            }
+            
+            @Step {
+                Open `Sources/App/Application+build.swift` again. We can now start the MongoKitten driver.
+                
+                This will connect and login to the server immediately. If the any network hiccups occur, MongoKitten reconnects automatically.
+                @Code(name: "Sources/App/Application+build.swift", file: mongokitten-setup-09.swift, previousFile: mongokitten-setup-06.swift)
+            }
+        }
+    }
+    
+    @Section(title: "Add Kittens API") {
+        @ContentAndMedia {
+            Add your database models and routes to edit them.
+            
+            @Image(source: "hummingbird.png", alt: "Hummingbird logo")
+        }
+        @Steps {
+            @Step {
+                Create a file named `Kitten.swift`, and add the following data model.
+                
+                MongoKitten always requires a stored property named `_id`. This is used by MongoDB as the unique ID.
+                @Code(name: "Sources/App/Kitten.swift", file: mongokitten-setup-10.swift)
+            }
+            
+            @Step {
+                If we look further down the file we can find the `buildRouter()` function.
+                
+                Here we create the `Router`. We add a logging middleware to it (this logs all requests to the router). The function uses a result builder to create a stack of middleware, but you can also use `Router.add(middleware:)` to add individual middleware. Finally we add a single endpoint GET `/` which returns "Hello!"
+                @Code(name: "Sources/App/Application+build.swift", file: mongokitten-setup-11.swift)
+            }
+            
+            @Step {
+                We'll add a single route `GET /kittens`, which lists all registered kittens.
```

Comment:
Mention something about passing the MongoDatabase to buildRouter(). Although given you don't need to add the database to the application you could create the database connection inside buildRouter().